packages/react/src/components/header/components/navigationLanguageSelector/NavigationLanguageSelector.tsx

```diff
@@ -35,34 +37,48 @@ const LanguageButton = ({ language }) => {
 
 const renderLanguageNode = (language: LanguageOption) => <LanguageButton key={language.value} language={language} />;
 
-export const NavigationLanguageSelector = ({ children, languages }: LanguageSelectorProps) => {
+export const NavigationLanguageSelector = ({
```

Comment:
I think the name "Navigation" is a bit redundant here as this under header components. Maybe just LanguageSelector?

Reply:
I think we should keep Navigation in the name - also other components under Header has Navigation/Header prefix in the name.